### DISCUSSION TITLE
How can I include other fragment from other files in Jinja2 template?

### DISCUSSION THREAD

**Discussion by joechiu-uipath** [microsoft/promptflow#1619] [Q&A] [answered]
I asked in another issue in General but no response there, so asking in Q&A. 
I have multiple prompts that would share some fragments of prompt text that I would like to manage centrally without duplicating.
So I want to have these fragment in their own file, and have the Jinja2 prompt files include them. 
Jinja2 syntax / engine supports include and import but they do not work out of box in promptflow, so I am wondering if that is supported or if there are alternate ways of doing this.

**Comment by brynn-code** [comment, marked as answer]
Let's discuss here, I think we could extract the same part in your prompt. :) #1566
